<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>yt-logo.ico</normaloff>yt-logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff;</string>
  </property>
  <widget class="QLineEdit" name="videoUrl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 7px;
border: 1px solid #ccc;</string>
   </property>
   <property name="placeholderText">
    <string>https://www.youtube.com/watch?v=QR4tJDcYCTY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="indirBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border: 1px solid #ccc;</string>
   </property>
   <property name="text">
    <string>Videoyu İndir</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxKalite">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Video Kalitesi</string>
   </property>
   <widget class="QRadioButton" name="radio480">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>480P</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio720">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>720P</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio1080">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1080P</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxFormat">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Video Formatı</string>
   </property>
   <widget class="QRadioButton" name="radioMp3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MP3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioMp4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MP4</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;a href=&quot;https://www.instagram.com/sametdrs3&quot; style=&quot;text-decoration:none; color: #000;&quot;&gt;Made By SD&lt;/a&gt;</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>42</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #fff;</string>
   </property>
   <property name="text">
    <string>SD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
